src/main/resources/project/ec_setup.pl

```diff
@@ -75,18 +75,18 @@
 
 # Data that drives the create step picker registration for this plugin.
 my %decommissionEnvironments = (
-	label     => "EF-Utilities - Decommission Environments",
-	procedure => "Decommission Environments",
+	label       => "EF-Utilities - Decommission Environments",
```

Comment:
Better 'Flow Utilities - Decommission Environments', same for 'Create Schedule To Trigger Gate' -> 'Flow Utilities - Create Schedule To Trigger Gate'